src/editor/Editor.js

```diff
@@ -221,6 +223,10 @@ define(function (require, exports, module) {
         description: Strings.DESCRIPTION_ALLOW_JAVASCRIPT
     });
 
+    PreferencesManager.definePreference(AUTO_UPDATE,         "boolean", true, {
```

Comment:
Other than this change, I don't think you need any of the other changes in this file so I would suggest removing them.